ui/apps/dev-server-ui/src/components/Header.tsx

```diff
@@ -17,7 +22,18 @@ export default function Header(props: Props) {
         {props.children}
       </nav>
       <div className="my-1 hidden md:block">
-        <AddAppButton />
+        {pathname === '/runs' ? (
+          <SendEventButton
+            label="Send test event"
+            data={JSON.stringify({
+              name: '',
+              data: {},
+              user: {},
+            })}
+          />
+        ) : (
+          <AddAppButton />
+        )}
```

Comment:
Happy to approve this, but I don't know where this will fit in when we move the dev server to the new navigation in the coming days. Perhaps a question for design

Reply:
When that time comes, I believe it will be moved to the initial location in the run designs. Right now, we agreed this was good enough to move forward.